src/main/java/walle/FileParser.java

```diff
@@ -0,0 +1,92 @@
+package walle;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileParser {
+    public static int readFileContents(String filePath, ArrayList<Task> tasks) throws FileNotFoundException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
```

Comment:
Consider using more intuitive naming, `f` and `s` do not explain what the variables entail.